testdata/picard/util/abutting.interval_list

```diff
@@ -0,0 +1,6 @@
+@HD	VN:1.6	SO:coordinate
+@SQ	SN:1	LN:249250621	AS:GRCh37	UR:http://www.broadinstitute.org/ftp/pub/seq/references/Homo_sapiens_assembly19.fasta	M5:1b22b98cdeb4a9304cb5d48026a85128	SP:Homo Sapiens
+1	30000	30001	+	target_1
+1	30002	30003	+	target_2
```

Comment:
perhaps use the name of the interval to comment about the properties of each interval:

e.g 

```
1	30000	30001	+	target_1_abutting_the_next
1	30002	30003	+	target_2_abutting_the_previous_but_not_the_next
1	30005	30007	+	target_3_overlapping_the_next_but_not_abutting_the_previous
1	30006	30008	+	target_4_overlapping_the_previous
```